components/FeedItemCard.vue

```diff
@@ -0,0 +1,119 @@
+          <template #placeholder>
+            <div class="d-flex align-center justify-center fill-height">
+              <v-container fluid class="fill-height">
+                <v-row class="ma-0">
+                  <v-col
+                    cols="12"
+                    class="pending font-weight-md-thin scale-text text-center"
+                  >
+                    {{ id }}
+                  </v-col>
+                  <v-col cols="12">
+                    <v-progress-linear
+                      indeterminate
+                      color="black"
+                      background-color="transparent"
+                      height="1"
+                      bottom
+                    />
+                  </v-col>
+                </v-row>
+              </v-container>
+            </div>
+          </template>
+          <template #error>
+            <v-container fluid class="fill-height">
+              <v-row class="ma-0">
+                <v-col cols="12">
+                  <div class="error404 scale-text text-center">
+                    404 image not found
+                  </div>
+                </v-col>
+              </v-row>
+            </v-container>
+          </template>
+          <template #default>
+            <v-fade-transition>
+              <div v-if="isHovering" class="hover fill-height pa-1 pl-4">
+                <div v-if="isPlayable()" id="playIcon">
+                  <v-icon>{{ 'mdi-play' }}</v-icon>
+                </div>
+                <v-row align="end" class="fill-height pa-1 pl-4">
+                  <v-col>
+                    <a class="white--text font-weight-bold">
+                      {{ data?.title || "Cool Artwork Piece" }}
+                    </a>
+                    <br>
+                    <a class="white--text font-italic">
+                      {{ data?.creator || "Cool Artist" }} 
+                    </a>
+                  </v-col>
+                </v-row>
+              </div>
+            </v-fade-transition>
+          </template>
+        </v-img>
+      </v-card>
+    </template>
+  </v-hover>
+</template>
+
+
+<script setup lang="ts">
+
+const hover = ref(false)
+const { id } = defineProps<{ id: string }>()
+const abc = useArtByCity()
+
+const { data, pending } = useLazyAsyncData(id, async () => {
+  const publication = await abc.legacy.fetchPublication(id)
+  console.log(publication)
+  return publication
+})
+
+function src() {
+  if (data.value) {
+    if ('images' in data.value && data.value.images.length > 0) {
+      return 'https://arweave.net/' + data.value.images[0].preview
+    }
+  }
+
+  return ''
+}
+
+function isPlayable():boolean {
+  if (data.value) {
+    if ('images' in data.value) {
+      return !!data.value.images[0].animated
+    } else if (
+      'audio' in data.value || 'model' in data.value
+    ) { return true }
+  }
+
+  return false
+}
+</script>
+
+
+<style scoped>
+
+.pending, .error404{
```

Comment:
you should use the vuetify helper classes instead of rolling your own each time you want to do font or color stuff.  there are a few reasons for doing it, like color switching out with theme swap, or consistent font sizing without copypasta.  see https://vuetifyjs.com/en/styles/text-and-typography/

so, remove these classes, and just use vuetify ones.  for size can use like `text-h2` or whatever size you think is best.  can change font weight with those helper classes and stuff too.  a lot of common things you'll want to do can be done easily with those vuetify helper classes.

for text color, shouldn't actually need to change it most of the time because the theme should handle it